pycode/memilio-epidata/memilio/epidata/getVaccinationData.py

```diff
@@ -427,7 +427,8 @@ def get_vaccination_data(read_data=dd.defaultDict['read_data'],
     vacc_column_names = [
         dd.EngEng['vaccPartial'],
         dd.EngEng['vaccComplete'],
-        dd.EngEng['vaccRefresh']]
+        dd.EngEng['vaccRefresh'],
+        'Vacc_refreshed_2']
```

Comment:
This is just a temporary replacement until Patricks pull request is finished, so that the CI isn't failing. I did not want to introduce another dict value that is only used temporary and afterwards not deleted.